Sources/Kitura/RouterResponse.swift

```diff
@@ -121,6 +121,20 @@ public class RouterResponse {
         statusCode = .unknown
     }
 
+    deinit {
+        if !state.invokedEnd {
+            if !state.invokedSend && statusCode == .unknown {
```

Comment:
I didn't say to remove the second if. Just o remove the `!state.invokeEnd &&` part.